libcaliptra/examples/hwmodel/interface.c

```diff
@@ -0,0 +1,125 @@
+//Licensed under the Apache-2.0 license
+
+#define HWMODEL 1
+
+#include <stdint.h>
+#include <stddef.h>
+#include <stdio.h>
+#include <stdbool.h>
+#include <errno.h>
+#include <unistd.h>
+
+#include "caliptra_model.h"
+#include "caliptra_api.h"
+
+#define CALIPTRA_STATUS_OK 0
+
+// Implementation specifics
+
+struct caliptra_model_init_params init_params;
+struct caliptra_fuses fuses = {0};
+struct caliptra_buffer image_bundle;
+
+static bool caliptra_model_init_complete = false;
+
+static struct caliptra_buffer read_file_or_die(const char* path)
```

Comment:
Suggest using a gentler function name :-)

Reply:
We'll get that in the next PR, this was adopted from the smoke test.